pkg/models/interval_action.go

```diff
@@ -21,10 +21,8 @@ import (
 )
 
 type IntervalAction struct {
+	BaseObject
```

Comment:
Not sure we should be adding BaseObject back in here. We want to get rid of it, as discussed, so I believe @xmlviking created this type with inline properties for that reason.

Reply:
Corrected.